src/content/docs/en/guides/cms/hygraph.mdx

```diff
@@ -0,0 +1,131 @@
+
+### Setting up the endpoint
+
+To add your Hygraph endpoint to Astro, create a `.env` file in the root of your project with the following variable:
+
+```ini title=".env"
+HYGRAPH_ENDPOINT=YOUR_HIGH_PERFORMANCE_API
+```
+
+Now, you should be able to use this environment variable in your project. 
+
+If you would like to have IntelliSense for your Contentful environment variables, you can create a `env.d.ts` file in the `src/` directory and configure `ImportMetaEnv` like this:
+
+```ts title="src/env.d.ts"
+interface ImportMetaEnv {
+  readonly HYGRAPH_ENDPOINT: string;
+}
+```
+
+:::note
+Read more about [using environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
+:::
+
+Your root directory should now include these new files:
+
+<FileTree title="Project Structure">
+- src/
+  - **env.d.ts**
+- **.env**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+### Fetching data
+
+Now, Astro components can fetch data from your Hygraph project by using the `HYGRAPH_ENDPOINT`. 
+
+For example, if you have multiple "blogPosts" content type that has a string field for a title:
+
+```astro title="src/pages/index.astro"
+---
+interface Post {
+	title: string;
+}
+
+const query = {
+	method: "POST",
+	headers: { "Content-Type": "application/json" },
+	body: JSON.stringify({
+		query: `
+      {
+        blogPosts {
+          title
+        }
+      }`,
+	}),
+};
+
+const response = await fetch(import.meta.env.HYGRAPH_ENDPOINT, query);
+const json = await response.json();
+const posts: Post[] = json.data.blogPosts;
+---
+
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<meta name="viewport" content="width=device-width" />
+		<meta name="generator" content={Astro.generator} />
+		<title>Astro</title>
+	</head>
+	<body>
+		<h1>Astro</h1>
+		{
+			posts.map((post) => (
+				<div>
+					<h2>{post.title}</h2>
+				</div>
+			))
+		}
+	</body>
+</html>
+```
+## Deploy
+
+If your project is using Astro's default static mode, you will need to set up a webhook to trigger a new build when your content changes. If you are using Netlify as your hosting provider, you can use its webhook feature to trigger a new build from Contentful events. 
+
+### Configuring Netlify Webhook
+
+To set up a webhook in Netlify:
+
+1. Deploy your site to Netlify with [this guide](/en/guides/deploy/netlify/). Remember to add your `HYGRAPH_ENDPOINT` to the enviroment variables.
+
+2. Then Go to your Hygraph project dashboard and click on **Apps**. 
+
+3. Go to the marketplace and search for Netlify and follow the instructions provided.
+
+4. If all went good, now you can deploy your website with a click in the Hygraph interface.
+
+## Example repo
+
+If you  want to see a working example of a Hygraph project with `marked` for markdown parsing, check out [this example repo](https://github.com/camunoz2/example-astrowithhygraph).
```

Comment:
We love to showcase community & official resources, so let's reorganise this under "Community Resources" to match our other guides.
```suggestion
## Community Resources

- [Hygraph + Astro example with `marked` for markdown parsing](https://github.com/camunoz2/example-astrowithhygraph)
```